gradle/plugins/src/main/kotlin/com.hedera.gradle.reports.gradle.kts

```diff
@@ -17,6 +17,7 @@
 plugins {
     id("jvm-ecosystem")
     id("jacoco-report-aggregation")
+    id("com.hedera.gradle.lifecycle")
```

Comment:
This adds in the `releaseMavenCentral` task which the `closeSonatypeStagingRepository` task depends on.

For this reports subproject `releaseMavenCentral` is a no-op.